### ISSUE TITLE
View/Edit Additional Language Details Information

### ISSUE THREAD

**Issue by sethmcknight** [SeedCompany/cord-field#374]
On the Language Details page we need to add the ability to view and edit the following information:

- [ ] All information that is currently available in the Edit modal should be displayed on the Details view (Details).
- [ ] One or more Locations should be able to be assigned to a Language (Details & Edit).
- [ ] A tag/chip to show that a Language is a Sign Language (Details & Edit).
- [ ] Sponsor Begin Date [calculated] and Sponsor End Year [input] (Details & Edit). 
- [ ] Language Sensitivity should be available to change in the Edit modal (Edit).

**Comment by CarsonF** [comment]
> * One or more Locations should be able to be assigned to a Language (Details & Edit).

We are waiting on that because locations are getting completely refactored right now

> * A tag/chip to show that a Language is a Sign Language (Details & Edit).

The needs to be added to the API

> * Sponsor Begin Date and Sponsor End Year (Details & Edit).

We currently only have the `sponsorDate` which is the earliest start date from its engagements. This is not editable because it's calculated from that.
Are you saying we also need a sponsor end date?

> * Language Sensitivity should be available to change in the Edit modal (Edit).

I thought we were basing sensitivity on the highest of the language's locations? As long as it's calculated it won't be editable.

**Comment by sethmcknight** [comment]
@CarsonF we also need to 
- Remove the 'Ethnologue ID' field (this is no longer used)
 
Language Profile Images
- Display on Language Card and Language Profile
- Default Language Images by Location (I can get these for you based on what's on sc.com)
- Allow users to upload a specific image to replace the default

**Comment by CarsonF** [comment]
I'd rather do images separately as it represents more work on the API and UI.

**Comment by sethmcknight** [comment]
@CarsonF sounds great, I'll break that off to its own issue. Is there anything else here that needs to be a separate issue or is the rest all just one issue?

**Comment by sethmcknight** [comment]
@frankyma thanks for getting this stuff added! I noticed in the PR we decided to push out Locations till the refactor is complete. Do you know if an issue exists for that? If not I'll get one created so we don't lose it.

**Comment by frankyma** [comment]
You're welcome 🙂.  There's no separate issue I'm aware of but I don't have a close eye on the entire issues list.

**Comment by sethmcknight** [comment]
We'll handle 'Locations' in #532
